rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

```diff
@@ -2,6 +2,32 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[june-26-2024]]
+== June 26, 2024
+
+// == What's New
+//include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| RPA Builder is now able to download runtime packages from RPA Manager. | W-16080683
+| Serialized JSON in an activity parameter of type string will not been modified during the execution of a process. | W-15881024
```

Comment:
```suggestion
| The first character of an Activity Parameter string values in JSON format no longer changes to lower case when passed from one Activity to another.
 | W-15881024
```

Reply:
Is this correct? Does this only happen for JSON format or for all strings or for all characters following a curly bracket?

Reply:
If we want to describe this in that detail I need to check it with the engineers.

Reply:
We only need as much detail as absolutely necessary for mentioning the behavior the users experienced (not the cause of it).
How about the following:
Activity Parameter [string] values [in JSON format] no longer change [to lower case] when passed from one Activity to another.
Please copy and remove all unnecessary parts.
Thank you.